### ISSUE TITLE
Hello, I ran 15 epochs with the code you gave, and the generated pth file was used to run the test set, but the test results were not as good as the pth path you gave, why is this?

### ISSUE THREAD

**Issue by yike5022** [GewelsJI/VPS#26]
Hello, I ran 15 epochs with the code you gave, and the generated pth file was used to run the test set, but the test results were not as good as the pth path you gave, why is this?

**Comment by GewelsJI** [comment]
Hi, @yike5022 

I assume you have settled everything down before running this code, like the code itself, and pre-trained weights. Also, could you send me the training logs here, so that we can look them over then?

**Comment by yike5022** [comment]
I can't find my training logs.So I may need to retrain, this question can be closed for now, and I'll ask you back when I get the results.Thank you for your answer.

**Comment by GewelsJI** [comment]
Surely. We can reproduce similar results according to our experience, but not sure what errors you encountered during your training.

This question is temporally closed

**Comment by Scatteredrain** [comment]
Hi, @yike5022 
Have you reproduced the similar results now? The same thing happened on my devices.

**Comment by yike5022** [comment]
I haven't tried it since, so please let me know if you have any new developments

**Comment by yike5022** [comment]
> Hi, @yike5022 Have you reproduced the similar results now? The same thing happened on my devices.

I haven't tried it since, so please let me know if you have any new developments
